lib/mpool/src/mclass.c

```diff
@@ -354,17 +351,17 @@ enum mpool_mclass
 mcid_to_mclass(enum mclass_id mcid)
 {
     switch (mcid) {
-        case MCID_CAPACITY:
-            return MP_MED_CAPACITY;
+    case MCID_INVALID:
+        return MP_MED_INVALID;
 
-        case MCID_STAGING:
-            return MP_MED_STAGING;
+    case MCID_CAPACITY:
+        return MP_MED_CAPACITY;
 
-        case MCID_PMEM:
-            return MP_MED_PMEM;
+    case MCID_STAGING:
+        return MP_MED_STAGING;
 
-        default:
-            break;
+    case MCID_PMEM:
```

Comment:
If you take away this case, does the compiler warn you? I am not sure if that warning is enabled by default in GCC with our current setup.

Reply:
Yes, it does warn.. I checked that.